snapcraft/project/_project.py

```diff
@@ -0,0 +1,29 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2018 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from ._project_options import ProjectOptions
+from ._project_info import ProjectInfo                 # noqa: F401
```

Comment:
Ah, that thread disappeared, I'll ask it here again:

I'm sorry, I should have made my question more clear, as it still stands: why is this imported at all? It's not used here.

Reply:
It is used as a type annotation in a comment here https://github.com/kalikiana/snapcraft/blob/c30a72598b6f4132d99523bc3eaf8a654a95587e/snapcraft/project/_project.py#L27

Reply:
Oh man... I'm blind, thank you for clarifying.